### ISSUE TITLE
Data schema stub showing up in posts list

### ISSUE THREAD

**Issue by emilianos** [TryGhost/gatsby-source-ghost#63]
### Issue Summary

Data schema stub showing up in posts list:
![image](https://user-images.githubusercontent.com/4334689/89124533-38c42c80-d4ae-11ea-86a2-41a529abb4d0.png)

I just upgraded to v3 of Ghost and updated all dependencies and noticed I have a post for "Data schema".
After digging a little bit and trying a clean install I could reproduce the same issue.

Also, this is visible in the demo at https://gatsby.ghost.org/
![image](https://user-images.githubusercontent.com/4334689/89124655-0a931c80-d4af-11ea-9820-76a3f68a30b6.png)

### To Reproduce

1. Do a clean install and it will show up

### Technical details:

* Ghost Version: 3.27.0
* Gatsby Version: 2.24.11
* Gatsby Source Ghost Version: 4.0.5
* Node Version: 10.18.1
* OS: MacOS 10.15.6

**Comment by jvkiran** [comment]
Facing Same issue.

**Comment by maxder** [comment]
Same issue. Can't seem to disable with `visibility: 'private',` in gatsby-source-ghost/ghost-schema.js.

**Comment by emilianos** [comment]
This is my temporal hack just in case:

`/src/components/common/PostCard.js`
```
 <Link to={url} className={"post-card " + url.slice(1, -1)}>
```
creates a classname with same name as url
then you can hide it with css:

`/src/styles/app.css`
```
.post-card.data-schema {
    display: none;
}
```

**Comment by jvkiran** [comment]
I've fixed by ternary operator

**Comment by maxder** [comment]
Brilliant. Both work great. I ended up with the ternary. But it looked a bit different in my index.js, author.js and tag.js, e.g.:

`const Index = ({ data, location, pageContext }) => {
    const posts = data.allGhostPost.edges

    return (
        <>
            <MetaData location={location} />
            <Layout isHome={true}>
                <div className="container">
                    <section className="post-feed">

                        {posts.map((value, i) => value.node.slug !== "data-schema" ? <PostCard key={value.node.id} post={value.node} /> : "" )}
                        
                    </section>
                    <Pagination pageContext={pageContext} />
                </div>
            </Layout>
        </>
    )
}`

**Comment by bobziroll** [comment]
Same issue here, and the Ghost Gatsby Starter on Ghost's own website is too. Ended up using a filter before my `.map()`, but still pretty hacky until this is fixed

[https://gatsby.ghost.org/](https://gatsby.ghost.org/)

**Comment by aileen** [comment]
Solved with #61
